src/layouts/Gallery.css

```diff
@@ -1,8 +1,14 @@
-.Gallery {
+/* This is to fix the size of the right section, .Gallery will take care of the flex and padding */
+.GalleryContainer {
     color: aliceblue;
-    width: 100vw;
+    width: calc(100% - 300px);
```

Comment:
hmm, so this is based on the sidebar? so if i change the sidebar width, i would have to change this too?

Reply:
Yeah.. css variables are a thing too but I'm too lazy to use them here

Reply:
will look into those! thanks! :D